quipucords/quipucords/tests_environment.py

```diff
@@ -16,6 +16,7 @@
 from django.test import TestCase
 
 from . import environment
+from . import release
```

Comment:
I don't think you need this import.

Reply:
Need it to implement your other suggestion